jobs/src/main/scala/dcos/metronome/model/SecretDef.scala

```diff
@@ -0,0 +1,28 @@
+package dcos.metronome.model
+
+/**
+  * A secret declaration
+  * @param source The source of the secrets value.
```

Comment:
this could use improvement... it really just repeats what is already known... the SecretDef is a secret declaration... and the param source is the source of a secret value.        There should be a lot more context here.   It should be clear what the source is.. a reference to a secret which will be injected with a value from the secret store.